### PR TITLE
Update sharepoint-online.md to add SPOSiteScriptPackge cmdlets

### PR DESCRIPTION
**Pull request by alinsirbu**
[PR for adding Add-SPOSiteScriptPackage](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/465)
[PR for adding Set-SPOSiteScriptPackage](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/466)

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ The following cmdlet references are for SharePoint Online.
 
 {{Manually Enter Add-SPOSiteScript Description Here}}
 
+### [Add-SPOSiteScriptPackage](Add-SPOSiteScriptPackage.md)
```
Comment (alinsirbu) [inline sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md:42]: https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/465

File: sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md [hunk 2]
```diff
@@ -463,6 +467,10 @@ This cmdlet is not currently active in production and may be removed in the futu
 
 {{Manually Enter Set-SPOSiteScript Description Here}}
 
+### [Set-SPOSiteScriptPackage](Set-SPOSiteScriptPackage.md)
```
Comment (alinsirbu) [inline sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md:470]: See [Pull Request](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/466)

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@alinsirbu 
Hi,

At your earliest convenience, please complete your account setup so that your GitHub and Microsoft accounts are connected:
- [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main)

Thank you!

**Comment by alinsirbu** [comment]
@tiburd thanks for the instruction on linking the 2 accounts - Done.

**Pull request merged**